testing/kokoro/system_tests.sh

```diff
@@ -87,31 +84,15 @@ if [ $GOLANG_SAMPLES_GO_VET ]; then
       git diff go.mod | tee /dev/stderr | (! read)
       [ -f go.sum ] && git diff go.sum | tee /dev/stderr | (! read)
       set +x
-
-      echo "Running 'goimports compliance check' in '$mod'..."
-      set -x
-      diff -u <(echo -n) <(goimports -d .)
-      set +x
     popd > /dev/null;
   done
 
-  # Generate a list of all go files not inside a go submodule.
-  # go vet throws an error if you run it and no files are found to process.
-  # If find has anything go vet will be run.
-  # Risk: Any unexpected find output could falsely trigger go vet.
+  # Always run 'go vet' from the root, which does not look at sub-modules.
```

Comment:
I don't recall "always running from the repo root" as an optional previously. Is this a testing process change to increase consistency? No objections.

Reply:
Increasing consistency. If `go vet` is broken anywhere, I'm OK blocking all new PRs.